apps/api/v2/src/ee/event-types/event-types_2024_06_14/outputs/create-phone-call.output.ts

```diff
@@ -0,0 +1,28 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { Type } from "class-transformer";
+import { IsEnum, IsString, ValidateNested } from "class-validator";
+
+import { SUCCESS_STATUS, ERROR_STATUS } from "@calcom/platform-constants";
+
+export class Data {
```

Comment:
I don't see it being imported anywhere. If thats the case better not to export it.